### DISCUSSION TITLE
New Packet Parsers

### DISCUSSION THREAD

**Discussion by infosechoudini** [libpnet/libpnet#558] [General]
What other packet parsers do we need?

**Comment by stappersg** [comment]
(consider that I didn't quite understand the question that opened this discussion)
[VLAN](#460)

**Reply by infosechoudini** [reply]
Sweet! We'll add more VLAN support to the list.

**Comment by infosechoudini** [comment]
I was also thinking of adding the following:

- DHCP
- BGP
- DNS
- OSPF
- FTP
- TELNET

**Reply by infosechoudini** [reply]
Plus more support for VLAN
